server/controllers/mapsController.js

```diff
@@ -0,0 +1,55 @@
+const axios = require("axios");
+
+const KEY = process.env.GOOGLE_MAPS_API_KEY;
+const urlHead = "https://maps.googleapis.com/maps/api"
```

Comment:
missing ;, but we also aren't actually using this?

Reply:
good catch, I'll remove it!